src/neo/Network/P2P/Message.cs

```diff
@@ -21,7 +20,7 @@ namespace Neo.Network.P2P
     /// <summary>
     /// Represents a message on the NEO network.
     /// </summary>
-    public class Message : ISerializable
+    public class Message
```

Comment:
I think that we don't need to lose `ISerializable` here, maybe it could be useful for plugins

Reply:
Done.